annotation_full.wdl

```diff
@@ -10,29 +10,39 @@ workflow annotation {
   String?  gm_license="/refdata/licenses/.gmhmmp2_key"
   Int     additional_threads=16
   String  container="microbiomedata/img-omics@sha256:d5f4306bf36a97d55a3710280b940b89d7d4aca76a343e75b0e250734bc82b71"
+  String  finish_ano_container = "microbiomedata/workflowmeta:1.1.1"
```

Comment:
Is there a reason we aren't using microbiomedata/img-omics@sha256:d5f4306bf36a97d55a3710280b940b89d7d4aca76a343e75b0e250734bc82b71?

Reply:
this was the workflow container listed below in the finish_ano section, so i brought it to the top. not sure why we are not using the main container